registry.k8s.io/images/k8s-staging-kas-network-proxy/images.yaml

```diff
@@ -52,7 +52,8 @@
     "sha256:0349e45ae85f9aec3fafc289498b0058b2c0c4725fe4e7f69fef361182976833": ["v0.29.0"]
     "sha256:41d71104e9d87dfe84895ddb5a38d13a27c761f74d8113f5af221fe2f0b7e1e2": ["v0.29.1"]
     "sha256:f796c2158024c2bd2f2cd61bc074f4f9ced83bc540dd55587567ac8d1066ce39": ["v0.29.2"]
-    "sha256:e5d4082225403e2c46ddcd55a951321fa8ed8922e458cc4b4d88000a6e240f99": ["v0.29.3"]
```

Comment:
don't remove this, it's not going to be deleted, and this just makes it harder to trace where it came from

(presubmit should really enforce this...)

instead, make sure your release notes cover the correct tags to use and maybe warn that this one is broken

Reply:
sure, i reverted it, so we can ignore the CI error and publish the same container image 0.29.4 as 0.29.3, right?

Reply:
> publish the same container image ANP 0.29.4 as ANP 0.29.3, right?

this sounds like the best option. You have some presubmit errors, maybe you can just add two tags in each entry? i.e.:

```
"sha256:...": ["v0.29.3", "v0.29.4"]
```

Reply:
you can add multiple tags to an image, I'm not sure if that makes sense in your case, but it is supported, as long as the tags are not mutated from what was already published